lib/register_screen.dart

```diff
@@ -0,0 +1,179 @@
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            const SizedBox(
+              height: 100,
+            ),
+
+            //Logoyu burda tanımladım.
+
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: Image.asset("images/img_ba.png"),
+            ),
+
+            //Register yazısını burada tanımladım.
+
+            const Padding(
+              padding: EdgeInsets.all(8.0),
+              child: Text("Register",
+                style: TextStyle(
+                    fontSize: 30,
+                    fontWeight: FontWeight.bold,
+                    color: Color(0xff191849)),
+              ),
+            ),
+
+            //Username butonunu burada tanımladım.
+
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: TextField(
+                controller: uncontroller,
+                decoration: InputDecoration(
+                  fillColor: const Color(0xffEEEEFE),
+                  filled: true,
+                  helperText: "Username",
+                  hintText: "Username",
+                  border: OutlineInputBorder(
+                    borderRadius: BorderRadius.circular(10),
+                  ),
+                ),
+              ),
+            ),
+
+            //Email butonunu burada tanımladım.
+
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: TextField(
+                controller: eccontroller,
+                decoration: InputDecoration(
+                  fillColor: const Color(0xffEEEEFE),
+                  filled: true,
+                  helperText: "E-Mail",
+                  hintText: "example@example.com",
+                  border: OutlineInputBorder(
+                    borderRadius: BorderRadius.circular(10),
+                  ),
+                ),
+              ),
+            ),
+
+            //Password butonunu burada tanımladım.
+
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: TextField(
+                controller: pccontroller,
+                decoration: InputDecoration(
+                  fillColor: const Color(0xffEEEEFE),
+                  filled: true,
+                  helperText: "Password",
+                  hintText: "Password",
+                  border: OutlineInputBorder(
+                    borderRadius: BorderRadius.circular(10),
+                  ),
+                ),
+              ),
+            ),
+
+            //Or register with butonunu burada tanımladım,altında sosyal medya logoları da bulunuyor.
+
+            const Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Text(
+                  "Or Register With",
+                  style: TextStyle(fontSize: 20),
+                ),
+              ],
+            ),
+            const SizedBox(
```

Comment:
Yazı fontları için bir paketimiz var onu seninle paylaşıncaya dek figmadaki boyutların aynısını yazmalısın henüz karşılaştıramadım ama eğer yazı boyutlarını rastgele girdiysen bunları tüm projede düzenlemelisin.